antenna-documentation/scripts/documentation_release.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+BASEDIR=$(readlink -f "$0")
+BASEDIR=$(dirname "$BASEDIR")
+
+set -e
+
+CURRENT_BRANCH=${git describe --tag}
+
+createReleaseDocumentation() {
+  chmod a+x createOrUpdateVersionDocumentationFolder.sh
+  sh ./createOrUpdateVersionDocumentationFolder.sh ${CURRENT_BRANCH}
```

Comment:
The `sh` prefix here overrides the shebang line in the script. So on systems where `sh` is not an alias for `bash`, you don't get Bash, but we want Bash, so you should omit the prefix.

Reply:
Similar below.